composer/metrics/nlp.py

```diff
@@ -265,11 +265,22 @@ def replace_underscore(text: str) -> str:
 
         return white_space_fix(remove_articles(handle_punc(lower(replace_underscore(answer))))).strip()
 
-    def update(self, outputs: List[str], labels: List[List[str]]):
+    def update(self, batch, outputs: List[str], labels: List[List[str]]):
```

Comment:
this is an API breaking change, so we need to make it backward compatible. I think you can do this by making batch the third argument and making it optional with a default of None. wdyt?

Reply:
I can do that but it's going to result in the HuggingFace model's update_metric method making little sense

Reply:
why does it make that method not make sense?